### ISSUE TITLE
Dashboard A11y: Detail Template View - Cannot focus or interact with preview pages 

### ISSUE THREAD

**Issue by BrittanyIRL** [GoogleForCreators/web-stories-wp#4202]
## Bug Description
You can't tab through the template page previews in detail view right now. 

## Expected Behaviour

Be able to interact with template detail view with keyboard. I would expect to be able to tab through pages in grid and use 'enter' to swap out the primary page on the screen. 

## Steps to Reproduce

Go to a detail template view, use keyboard to move focus with tab and see that the template preview pages are skipped over.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

- fix should include karma tests

**Comment by csossi** [comment]
Verified in QA

**Comment by o-fernandez** [comment]
This is now possible, so closing the ticket. But, the "page selected" indicator and "keyboard focus" outline are too similar. We'll need to update this in a future ticket. #4418
